Sources/BraintreeCore/BTClientToken.swift

```diff
@@ -3,7 +3,8 @@ import Foundation
 /// An authorization string used to initialize the Braintree SDK
 @objcMembers public class BTClientToken: NSObject, NSCoding, NSCopying {
 
-    // TODO: resolve public access modifiers once rest of core is in Swift
+    // NEXT_MAJOR_VERSION (v7): properties exposed for Objective-C interoperability.
+    // Once the entire SDK is in Swift, determine if we want public properties to be internal
```

Comment:
Should the docstrings below get a `nodoc` if not intended for merchants?

Reply:
In `5.x` everything here was also in the [BTClientToken.h public header](https://github.com/braintree/braintree_ios/blob/5.x/Sources/BraintreeCore/Public/BraintreeCore/BTClientToken.h). Though it looks like most of the properties may have only been public to expose them to other Obj-C modules. This is more-so just to denote that we should explore if these need to remain public or if it was mostly done for interoperability reasons.

Seems like the [ClientToken class in Android](https://github.com/braintree/braintree_android/blob/master/BraintreeCore/src/main/java/com/braintreepayments/api/ClientToken.java) has these properties as internal, so may be able to clean up public access once we don't need to worry about Obj-C interoperability.

Reply:
Ah gotcha. 

I would guess that this entire file was just exposed for Drop-In. We never instruct merchants to interact directly with `BTClientToken`, they just pass us a raw string into the `BTAPIClient`. 

I can't see any case where we'd want a merchant to use these values, or even instantiate a `BTClientToken` at all - so some `nodoc`s probably couldn't hurt. What do you think?

Reply:
Ah, that makes sense that it's exposed for the Drop-in. Yep we can add nodoc for this and specifically call out that it's exposed for the Drop-in as well. Will get this updated. 🚀

Reply:
It looks like some of these are used in the Drop-in and others not - still worth adding a `nodoc` but added an additional note about evaluating this entire class at some point down the road.